cmd/podman/system/prune.go

```diff
@@ -137,7 +154,7 @@ func createPruneWarningMessage(pruneOpts entities.SystemPruneOptions) string {
 	}
 	return `WARNING! This command removes:
 	- all stopped containers
-	- all networks not used by at least one container%s%s
+	- all networks not used by at least one container%s%s` + pinnedNote + `
```

Comment:
No need to conditionalize this IMO - just say "optionally excluding" instead of "excluding". We can trust folks to read the help text, I think.

Reply:
Done. Removed the conditional logic and changed to a static "(excluding pinned volumes)" message that's always displayed.